Signal/src/ViewControllers/MediaMessageView.swift

```diff
@@ -308,22 +315,20 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
 
     // MARK: - OWSAudioAttachmentPlayerDelegate
 
-    public func isAudioPlaying() -> Bool {
-        return isAudioPlayingFlag
-    }
-
-    public func setIsAudioPlaying(_ isAudioPlaying: Bool) {
-        isAudioPlayingFlag = isAudioPlaying
-
-        updateAudioStatusLabel()
+    public func audioPlaybackState() -> AudioPlaybackState {
+        return playbackState
     }
 
-    public func isPaused() -> Bool {
-        return isAudioPaused
+    public func setAudioPlaybackState(_ value: AudioPlaybackState) {
```

Comment:
I'd love to, but I can't figure out how to get it to compile since we're trying to conform to a obj-c protocol.  Please feel free to take a crack at it!